src/Sulu/Bundle/MediaBundle/Tests/Unit/Media/Manager/MediaManagerTest.php

```diff
@@ -210,6 +221,24 @@ public function testDelete()
         $this->mediaManager->delete(1, true);
     }
 
+    public function testSpecialCharacterFileName()
+    {
+        $fileName = 'aäüßa';
+
+        /** @var UploadedFile $uploadedFile */
+        $uploadedFile = $this->prophesize(UploadedFile::class)->willBeConstructedWith(['', 1, null, null, 1, true]);
+        $uploadedFile->getClientOriginalName()->willReturn($fileName);
+        $uploadedFile->getPathname()->willReturn('');
+        $uploadedFile->getSize()->willReturn('123');
+        $uploadedFile->getMimeType()->willReturn('img');
+
+        $user = $this->prophesize(User::class)->willImplement(UserInterface::class);
+        $this->userRepository->findUserById(1)->willReturn($user);
+
+        $this->pathCleaner->cleanup(Argument::cetera($fileName))->shouldBeCalled();
```

Comment:
`Argument::cetera` means that you can pass anything, I doubt that this is what you want here. Just write `cleanup($fileName)` should be fine.